app/views/layouts/_primary_navigation.html.erb

```diff
@@ -23,16 +23,16 @@
 
           <li class="moj-primary-navigation__item">
             <%= link_to t('primary_navigation.open_apps'),
-                        crime_applications_path(status: 'open'),
+                        open_crime_applications_path,
```

Comment:
Other than the path these are identical? Could DRY it up - not sure if its worth it though unless we use it elsewhere?